ethereum_history_api/circuits/lib/src/misc/iterable.nr

```diff
@@ -0,0 +1,3 @@
+trait Iterable<Env, TItem> {
+  fn each(self, callback: fn[Env] (TItem) -> ()) -> ();
```

Comment:
Discussed offline that this is to be removed for now as lambdas are not functional now as we see in tests